src/Features/LanguageServer/Protocol/Handler/SemanticTokens/SemanticTokensHelpers.cs

```diff
@@ -0,0 +1,293 @@
+                [ClassificationTypeNames.XmlDocCommentAttributeQuotes] = LSP.SemanticTokenTypes.Comment, // TO-DO: Potentially change to custom type
+                [ClassificationTypeNames.XmlDocCommentAttributeValue] = LSP.SemanticTokenTypes.Comment, // TO-DO: Potentially change to custom type
+                [ClassificationTypeNames.XmlDocCommentCDataSection] = LSP.SemanticTokenTypes.Comment, // TO-DO: Potentially change to custom type
+                [ClassificationTypeNames.XmlDocCommentComment] = LSP.SemanticTokenTypes.Comment, // TO-DO: Potentially change to custom type
+                [ClassificationTypeNames.XmlDocCommentDelimiter] = LSP.SemanticTokenTypes.Comment, // TO-DO: Potentially change to custom type
+                [ClassificationTypeNames.XmlDocCommentEntityReference] = LSP.SemanticTokenTypes.Comment, // TO-DO: Potentially change to custom type
+                [ClassificationTypeNames.XmlDocCommentName] = LSP.SemanticTokenTypes.Comment, // TO-DO: Potentially change to custom type
+                [ClassificationTypeNames.XmlDocCommentProcessingInstruction] = LSP.SemanticTokenTypes.Comment, // TO-DO: Potentially change to custom type
+                [ClassificationTypeNames.XmlDocCommentText] = LSP.SemanticTokenTypes.Comment, // TO-DO: Potentially change to custom type
+                [ClassificationTypeNames.XmlLiteralAttributeName] = LSP.SemanticTokenTypes.Comment, // TO-DO: Potentially change to custom type
+                [ClassificationTypeNames.XmlLiteralAttributeQuotes] = LSP.SemanticTokenTypes.Comment, // TO-DO: Potentially change to custom type
+                [ClassificationTypeNames.XmlLiteralAttributeValue] = LSP.SemanticTokenTypes.Comment, // TO-DO: Potentially change to custom type
+                [ClassificationTypeNames.XmlLiteralCDataSection] = LSP.SemanticTokenTypes.Comment, // TO-DO: Potentially change to custom type
+                [ClassificationTypeNames.XmlLiteralComment] = LSP.SemanticTokenTypes.Comment, // TO-DO: Potentially change to custom type
+                [ClassificationTypeNames.XmlLiteralDelimiter] = LSP.SemanticTokenTypes.Comment, // TO-DO: Potentially change to custom type
+                [ClassificationTypeNames.XmlLiteralEmbeddedExpression] = LSP.SemanticTokenTypes.Comment, // TO-DO: Potentially change to custom type
+                [ClassificationTypeNames.XmlLiteralEntityReference] = LSP.SemanticTokenTypes.Comment, // TO-DO: Potentially change to custom type
+                [ClassificationTypeNames.XmlLiteralName] = LSP.SemanticTokenTypes.Comment, // TO-DO: Potentially change to custom type
+                [ClassificationTypeNames.XmlLiteralProcessingInstruction] = LSP.SemanticTokenTypes.Comment, // TO-DO: Potentially change to custom type
+                [ClassificationTypeNames.XmlLiteralText] = LSP.SemanticTokenTypes.Comment, // TO-DO: Potentially change to custom type
```

Comment:
we should have a tracking issue that tracks what we're doign to do here about having the rich set of colors for lsp.

Reply:
Just opened up an issue on our side here: https://github.com/dotnet/roslyn/issues/46582